src/configuration/services/kafka.md

```diff
@@ -6,7 +6,11 @@ See the [Kafka documentation](https://kafka.apache.org/documentation/) for more
 
 ## Supported versions
 
-* 2.1
+<div id = "kafkaSupported"></div>
+
+<script>
+makeImagesList("services", "kafka", "supported", "kafkaSupported");
```

Comment:
Smart :)
Did you do the same with languages services?

Reply:
Yes, all the languages and services have this function to fill the "Supported" and, if they have them, "Deprecated" sections.